### ISSUE TITLE
Add annotation types from ANNOTATION opcode

### ISSUE THREAD

**Issue by rocky** [rocky/python-decompile3#4]
## Description

Show annotation types in function signatures

## Background

Right now we only show annotation types for return values of functions. Showing them for the parameters is straight-forward. Changes should only be needed in the `semantics` directory. 

We have code to handle earlier (3.3 or so) annotations.See the annotation parameter in [`MAKE_FUNCTION`](https://docs.python.org/3/library/dis.html?highlight=annotation#opcode-MAKE_FUNCTION).  [unpy37](ttps://github.com/andrew-tavera/unpyc37/) I believe handles this correctly too. 

## Tests
Decomping a compiled version of: 

```
def foo(a, b: 'annotating b', c: int) -> float:
    print(a + b + c)
```

should show parameters `b`, and `c` annotated with their types.

**Comment by rocky** [comment]
@x0ret based on the kinds o f things you have been doing recently - thorough investigation of some of the gaps in the code, I thought this might be the kind of thing that you might both be interested in and good at doing. 

If however this is not of interest, let me now and I'll remove the assignment (or I think you can do that as well.)

**Comment by rocky** [comment]
https://github.com/rocky/python-uncompyle6/issues/169 is the same thing in uncompyle6.

**Comment by rocky** [comment]
Fixed by #6
